packages/rum-core/test/common/utils.spec.js

```diff
@@ -70,19 +70,6 @@ describe('lib/utils', function() {
     html.removeChild(script)
   })
 
-  it('should getNavigationTimingMarks', function() {
-    var marks = utils.getNavigationTimingMarks()
-    expect(marks.fetchStart).toBeGreaterThanOrEqual(0)
-    expect(marks.domInteractive).toBeGreaterThanOrEqual(0)
-    expect(marks.domComplete).toBeGreaterThanOrEqual(0)
-    expect(marks.loadEventEnd).toBeGreaterThanOrEqual(0)
-  })
-
-  it('should getPaintTimingMarks', function() {
-    var marks = utils.getPaintTimingMarks()
-    expect(marks).toEqual({})
-  })
```

Comment:
Did you mean to move these tests? or do you think they are not necessary anymore?

Maybe we should replace these with some improved tests, WDYT?

Reply:
we already have the test for FCP and agent marks in captureNavigation. Having this test doesn't provide any value and we have to expose that function just for the tests, so removed it.